README.md

```diff
@@ -1,2 +1,2 @@
 # pos.webapp
-asantaes app
+this is changed now
```

Comment:
you should make this a proper sentence ie uppercase first letter and a period at the end